src/templates/tombstone.jsx

```diff
@@ -0,0 +1,54 @@
+import {graphql} from 'gatsby';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Layout from '../layout';
+import SEO from '../components/SEO';
+import WarningsIcon from '../components/WarningsIcon';
+
+function TombstonePage({data: {suspendedPlugin: plugin}}) {
+    const pluginPage = 'templates/plugin.jsx';
+    return (
+        <Layout id="pluginPage" reportProblemRelativeSourcePath={pluginPage} reportProblemUrl={`/${plugin.id}`} reportProblemTitle={plugin.id}>
+            <SEO title={plugin.id} description={`Deprecated plugin: ${plugin.id}`} pathname={`/${plugin.id}`}/>
+
+            <div className="container">
+                <div className="title-wrapper">
+                    <h1 className="title">
+                        {plugin.id}
+                    </h1>
+                    <div className="plugin-id">
+                        {'ID: '}
+                        {plugin.id}
+                    </div>
+                </div>
+                <div className="alert alert-warning alert-with-icon">
+                    <WarningsIcon/>
+                    {'Plugin was removed from distribution, see '}
```

Comment:
This would align with the wording I generally use when talking about this. Seems less final (which in many cases it is not).

```suggestion
                    {'Plugin distribution has been suspended, see '}
```